docs/ScalarDLBackupCreationRestoration.md

```diff
@@ -0,0 +1,90 @@
+# Scalar DL Backup Creation on Kubernetes
+
+This guide shows you how to create and restore Scalar DL backups on Cosmos DB and DynamoDB without data inconsistencies.
+
+## Prerequisites
+
+* Read the Scalar DL backup creation and restoration guide, [see](https://github.com/scalar-labs/scalardl/blob/master/docs/backup-restore.md)
+* Scalar DL Ledger (and Auditor) must be deployed in the Kubernetes cluster
+* Cosmos DB account must be created with the backup policy `continuous` if you use Cosmos DB
+* DynamoDB tables must be created with point-in-time recovery (by default, scalardl-schema-loader enables PITR)
+
+## Backup Creation
+
+This section shows how to create a transactionally-consistent backup for Scalar DL.
+
+### Requirements
+
+* You must enable point-in-time recovery/restore in the backend database.
```

Comment:
Is it saying the same thing as above in a different way? 
Very redundant and should be united.

Reply:
Sorry, It is missed.
I missed it because the same idea described different sentences.

I will update the document without repeating the concept.